packages/documentation/src/stories/accessibility-practices/foundational-structure-and-semantics/reference-crossing-the-shadowdom/aria-describedby/examples/ariadescribedbyElementsProperty.sample.html

```diff
@@ -0,0 +1,3 @@
+<element id="elementId"></element>
+
+<span id="descriptionId"></span>
```

Comment:
I would add a little content (text) to both of these elements.

Is the `<element>` tag representing a custom element?
If yes, it needs to have a dash in the name `<my-element>`.
If no, why not using a native element like `<div>` or so?